src/components/index.ts

```diff
@@ -55,3 +55,4 @@ export { SelectListProps } from './SelectList/types';
 export { Row, RowProps, Column, ColumnProps } from './Grid/Grid';
 export { Popover, PopoverProps } from './Popover/Popover';
 export { InfoBanner, InfoBannerProps, InfoBannerVariants } from './InfoBanner/InfoBanner';
+export { InfoBannerCard } from './InfoBanner/blocks/InfoBannerCard';
```

Comment:
How does the import look when using the library?

Reply:
being exported here, it should be from `import { InfoBannerCard } from '@freenow/wave'` right?

Reply:
As said below, happy to refine the implementation and export strategy later, I implemented most of the changes from MR suggestions apart from touching the original API and export here.